prdoc/pr_5546.prdoc

```diff
@@ -0,0 +1,33 @@
+title: "Transfer polkadot native assets to Ethereum through snowbridge"
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Transfer polkadot native asset to Ethereum through snowbridge.
+
+crates:
+  - name: snowbridge-pallet-inbound-queue
+    bump: patch
+  - name: snowbridge-pallet-inbound-queue
+    bump: patch
+  - name: snowbridge-pallet-outbound-queue
+    bump: patch
+  - name: snowbridge-pallet-system
+    bump: major
+  - name: snowbridge-core
+    bump: major
+  - name: snowbridge-router-primitives
+    bump: major
```

Comment:
https://github.com/paritytech/polkadot-sdk/pull/5546/commits/5be8f8224af731e551671cb5f7412e90b9413e46

Reply:
In theory the change is `major` since you are adding new functions to public interfaces, but in practice we can also do only `minor` since the changes don't cause any backward compatibility issues.

The CI semver check cannot reason that deep, it sees public API modified so it asks for `major`. When we, the engineers, know better and want to override it, we can do:

```
name: snowbridge-core
   patch: minor
   validate: false
```

See more details [here](https://forum.polkadot.network/t/psa-polkadot-sdk-to-use-semver/7616).

Reply:
https://github.com/paritytech/polkadot-sdk/pull/5546/commits/de6d3ee61759711c10f4a88d4c8eaffe78edf1d7